packages/reports/app/styles/navi-reports/components/grouped-list.scss

```diff
@@ -4,8 +4,10 @@
  */
 .grouped-list {
   flex: 1;
+  margin-right: -10px;
```

Comment:
Do we need negative?

Reply:
Added a comment, this is a fix for the scrollbar covering the filter buttons in column-selector